build.sh

```diff
@@ -15,6 +15,7 @@ rm -rf $P/scratch;
 mkdir -p $P/scratch;
 cd $P/scratch;
 rm -rf $P/install;
-$SRC/configure --prefix=$P/install --with-mountutildir=$P/install/sbin --with-initdir=$P/install/etc --localstatedir=/var --enable-bd-xlator=yes --enable-debug --silent
-make install CFLAGS="-g -O0 -Wall -Werror" -j 4 >/dev/null
+$SRC/configure --prefix=$P/install --with-mountutildir=$P/install/sbin
+--with-initdir=$P/install/etc --localstatedir=/var --enable-bd-xlator=no --enable-debug --silent
```

Comment:
Isn't this  jumping to a new line ?
I suppose you should clip the above line with backward slash '\'